### ISSUE TITLE
Icon conditions on card title

### ISSUE THREAD

**Issue by carlywarly** [marcokreeft87/room-card#28]
Like the card, it's helping me tidy up by dashboard.  Would to be possible to add state_color to the icon on the title of the card?  For example, highlight icon next to Hall, if the hall light is on?  

![image](https://user-images.githubusercontent.com/50866001/188991271-b413f85d-b0c7-4e33-b1ad-f3485a3712cd.png)

I was hoping to use the icon to show occupancy :) Possibly add condition to change the icon as well?
```yaml
type: custom:room-card
title: Office
entity: input_boolean.inoffice
state_color: true
state_show: true
icon:
  conditions:
    - icon: mdi:door-open
      condition: equals
      value: 'off'
    - icon: mdi:door-closed
      condition: equals
      value: 'on'
entities:
  - entity: switch.den_light
    icon: phu:ceiling-round
    name: Light
    tap_action:
      action: toggle
    state_color: true
```

cheers,
Carl

**Comment by marcokreeft87** [comment]
Hey Carl, 
I think you found a bug regarding to the icon. That should be fixed in 1.2.6. 
The icon should be highlighted when the entity is active.

**Comment by carlywarly** [comment]
That was quick, many thanks :)
